src/components/MakeWidget/subcomponents/ActionButtons/ActionButtons.styles.tsx

```diff
@@ -23,8 +23,12 @@ const ButtonStyle = css`
 
 export const BackButton = styled(Button)`
   ${ButtonStyle}
+
+  padding-top: 0.7rem;
```

Comment:
```suggestion
```

The underlying problem is firefox alignment issue, there's a separate ticket for this: 
https://github.com/orgs/airswap/projects/8/views/1